lib/kafo/kafo_configure.rb

```diff
@@ -144,6 +144,7 @@ def initialize(*args)
       parse clamp_app_arguments
       parse_app_arguments # set values from ARGS to config.app
       Logging.setup(verbose: config.app[:verbose]) unless ARGV.any? { |option| ['--help', '--full-help'].include? option }
+      logger.notice("Loading installer configuration. This will take some time...")
```

Comment:
My feeling is we can do without the ellipsis.  And this just be a statement.

Reply:
Ok, that works for me.